tests/Shared/GetTestProjects.proj

```diff
@@ -1,48 +0,0 @@
-<Project DefaultTargets="GenerateListOfTestsForGithubActions">
```

Comment:
If this is being dropped then the command line in `.github/actions/enumerate-tests/action.yml` needs to be updated.

Reply:
Opps, this was late at night. I was meant to delete SolutionTests.proj.
Thanks